calcom/cal.com
# test: Integration Test GCal Primary Calendar

## What does this PR do?



This PR aims to add E2E testing to our GCal integration. The tests use a predefined QA user that is already connected to Google. When going through this PR is to think about what will happen when 10 of these tests are running at the same time.

Next step is to add tests for:
- Ensure that events on external calendars are blocked on Cal.com
- Tests for non primary calendars
- Expand this to Outlook



Fixes # (issue)



## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change





## How should this be tested?



- Set the `.env` variables `E2E_TEST_CALCOM_QA_EMAIL` and `E2E_TEST_CALCOM_QA_PASSWORD`
- Create a QA user account with the above .env variables
- Connect a Google account to this QA user
- Run the tests found in `packages/app-store/googlecalendar/tests/google-calendar.e2e.ts`
- If you can, add breakpoints to certain steps of the tests to ensure that the events are being created on GCal

## Mandatory Tasks


## Checklist
